engine/src/flutter/impeller/compiler/shader_lib/impeller/rrect.glsl

```diff
@@ -0,0 +1,44 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef RRECT_GLSL_
+#define RRECT_GLSL_
+
```

Comment:
This needs the include for math.glsl for POW IIRC. It might just work because includes are just text concatenation.